src/network/netlink.rs

```diff
@@ -212,16 +212,22 @@ impl Socket {
         Ok(())
     }
 
-    fn create_route_msg(route: &Route) -> RouteMessage {
+    fn create_route_msg(route: &Route, metric: Option<u32>) -> RouteMessage {
         let mut msg = RouteMessage::default();
 
+        let final_metric = metric.unwrap_or(100);
```

Comment:
Can please make a comment why 100 is the default vs not setting it in the netlink call and let the kernel choose?
I assume this is because the kernel uses 0 as default which makes it highest prio?